### PR TITLE
updated futures page

### PR DESCRIPTION
**Pull request by jmzwar**
Screen Shot 2022-12-13 at 11 23 34 am

### CODE REVIEW

File: src/components/EmailSignup/Wrapper.tsx [hunk 1]
```diff
@@ -1,11 +1,17 @@
-import { Box, Flex, BoxProps } from '@chakra-ui/react';
+import { Box, Flex, FlexProps } from '@chakra-ui/react';
 import { ReactNode } from 'react';
 
-interface WrapperProps extends BoxProps {
+interface WrapperProps extends FlexProps {
```
Comment (bachstatter) [inline src/components/EmailSignup/Wrapper.tsx:4]: hmm I guess this should still extend  BoxProps since we're spreading it on a box?

Comment (bachstatter) [reply src/components/EmailSignup/Wrapper.tsx:4]: Not sure if the types are different though

File: src/sections/futures/frontends.tsx [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+						Want to integrate Synthetix Perps with your Platform?
+					</Text>
+					<Text mt={1} color="gray" fontSize="12px" lineHeight="18px">
+						Learn more about how to integrate Perps and other Synthetix markets
+						in our developer documentation and reach out to the team at
+						<Text as="span"> perps@snxdao.io</Text>
+					</Text>
+				</Box>
+				<Link
+					href="https://www.imdb.com/title/tt0209163/"
```
Comment (bachstatter) [inline src/sections/futures/frontends.tsx:201]: Worth having a todo here

File: src/components/EmailSignup/Wrapper.tsx (hunk 1 above)
Comment (jmzwar) [reply src/components/EmailSignup/Wrapper.tsx:4]: Minor differences.

File: src/sections/futures/frontends.tsx (hunk 2 above)
Comment (jmzwar) [reply src/sections/futures/frontends.tsx:201]: Good point

**Comment by bachstatter** [comment]
Few things:
- Did you ask anyone if it's suppose to go into the header nav?
- "Trading Perpetuals", worth if the product name is Perps, maybe we want to use that 
- The stats section in the middle need to either be removed, or start using kwenta's subgraph. https://thegraph.com/hosted-service/subgraph/kwenta/optimism-main
- For the "Trading Perpetuals" section, do we want to show markets from futures v1?
- We also need to make sure that Perpsv2 markets will show up

**Comment by jmzwar** [comment]
> Few things:
> 
> * Did you ask anyone if it's suppose to go into the header nav?
> * "Trading Perpetuals", if the product name is Perps, maybe we want to use that
> * The stats section in the middle need to either be removed, or start using kwenta's subgraph. https://thegraph.com/hosted-service/subgraph/kwenta/optimism-main
> * For the "Trading Perpetuals" section, do we want to show markets from futures v1?
> * We also need to make sure that Perpsv2 markets will show up

Few changes - switched from /futures to /perps
=> linking to discord for integration
=> Added perps v1/perps v2 markets
=> Hiding stats

File: src/sections/perps/perpetuals.tsx [hunk 3]
```diff
@@ -0,0 +1,190 @@
+import { useState } from 'react';
+import { Flex, FlexProps, Text, Button, Link } from '@chakra-ui/react';
+import { PerpetualSynth } from 'pages/perps';
+import FutureSynthCard from 'src/components/FutureSynthCard';
+
+interface PerpetualsProps extends FlexProps {
+	synths: PerpetualSynth[];
+}
+
+const perpV2Synths = ['sETH'];
```
Comment (bachstatter) [inline src/sections/perps/perpetuals.tsx:10]: Maybe add a todo saying we need to figure out what futures are live in a better way

File: src/sections/perps/perpetuals.tsx [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+	const [activeTab, setActiveTab] = useState('all');
+	const [activeVersion, setActiveVersion] = useState<'V1' | 'V2'>('V1');
+
+	let tabs: string[] = ['all'];
+
+	const synthsDictionaryV1: Record<string, JSX.Element[]> = {
+		all: [],
+	};
+
+	const synthsV2: PerpetualSynth[] = [];
```
Comment (bachstatter) [inline src/sections/perps/perpetuals.tsx:22]: Feels a little wrong to create the dictionaries in the render fn, I would extract to a function. But I guess you just followed the already existing patterns

**Review by bachstatter — APPROVED** [review]

File: src/sections/perps/perpetuals.tsx (hunk 4 above)
Comment (jmzwar) [reply src/sections/perps/perpetuals.tsx:22]: The synths are being passed as a prop - I think once we have a list for both we can update this pattern.

File: src/sections/perps/perpetuals.tsx (hunk 3 above)
Comment (jmzwar) [reply src/sections/perps/perpetuals.tsx]: Yup done

**Pull request merged**